src/TesterInternal/StorageTests/PersistenceGrainTests_AzureStore.cs

```diff
@@ -37,12 +37,18 @@ public class PersistenceGrainTests_AzureStore : UnitTestSiloHost
         private const int MaxReadTime = 200;
         private const int MaxWriteTime = 2000;
 
+        private static readonly Guid initialServiceId = Guid.NewGuid();
+
         private static readonly TestingSiloOptions testSiloOptions = new TestingSiloOptions
         {
             SiloConfigFile = new FileInfo("Config_AzureTableStorage.xml"),
```

Comment:
You are right, this got messed up somehow. I'm adding the file back to the csproj.